src/ophys_etl/modules/denoising/fine_tuning/schemas.py

```diff
@@ -15,6 +23,12 @@ class DataSplitterInputSchema(argschema.ArgSchema):
         description='Amount of data to set aside for training. The rest will '
                     'be used for validation'
     )
+    downsample_frac = argschema.fields.Float(
+        default=0.0,
+        description='Amount to downsample the data by. I.e. a downsample frac '
+                    'of 0.1 would reduce the dataset by 10%'
```

Comment:
Not sure what it is, but for some reason I would have thought downsampling would be we sample 10% of the data versus remove 10% of the data. Maybe add slightly more description here.